### ISSUE TITLE
[DevOps] Discovery on `checkin-devops` repository and contents

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-team#77265]
## User Story
As a DevOps engineer, I'd like to know what's in [`checkin-devops`](https://github.com/department-of-veterans-affairs/checkin-devops/) so that I can maintain and, where possible, improve it.

## Tasks
- [x] [DevOps] Check out `checkin-devops` and carefully read through its contents.

## Acceptance Criteria
- [ ] `checkin-devops` has been 👀ed and findings have been documented in this thread.
- [ ] Followup issue(s) opened if appropriate to take action where advisable.

**Comment by ndouglas** [comment]
- general:
  - setup Dependabot updates for GHA
- `.github/workflows/day-end-summary.yml`:
  - pin action versions
  - specify requirements in `requirements.txt`
  - tweak formatting
- `.github/workflows/ecs-cron-redeploy.yml`:
  - pin action versions
  - does this need to be done sequentially? (frontend then backend)
  - notify Slack on failure
- `.github/workflows/ecs-redeploy.yml`:
  - pin action versions
  - shift cycling logic into a shared action
  - notify Slack on failure
- `.github/workflows/terraform-apply.yml`:
  - consider factoring out Slack notification
  - pin action versions
- `.github/workflows/terraform-caller-apply.yml`:
  - pin action versions
- `.github/workflows/terraform-caller-plan.yml`:
  - pin action versions

**Comment by ndouglas** [comment]
Some of this is blocked by onboarding tasks I still need to complete (`vaec-cms` AWS account, GFE, etc).

**Comment by ndouglas** [comment]
This is no longer blocked.
